share/spice/similar_sites/similar_sites.js

```diff
@@ -0,0 +1,63 @@
+(function(env) {
+
+    env.ddg_spice_similar_sites = function(api_result) {
+
+        var script = $('[src*="/js/spice/similar_sites/"]')[0],
+                source = $(script).attr("src"),
+                query = source.match(/similar_sites\/([^\/]+)/)[1];
+
+        if(!api_result || !api_result.num === 0) {
+            return Spice.failed('similar_sites');
+        }
+
+        var num = api_result.num;
+        var show_more = true;
+        if (num < 5)
+            show_more = false;
+
+        delete api_result['num'];
+        delete api_result['status'];
+
+        Spice.add({
+            id: "similar_sites",
+            name: "Similar Sites",
+
+            data: {
+                results: api_result,
+                num: num,
+                show_more: show_more,
+                show_more_count: num-5
+            },
+
+            meta: {
+                total: num,
+                sourceName: 'SimilarSites',
+                sourceUrl: 'http://www.similarsites.com/site/' + query,
+                sourceIcon: true,
+                itemType: 'Similar Sites'
+            },
+
+            templates: {
+                group: 'base',
+                options: {
+                    content: Spice.similar_sites.content
+                }
+            }
+        });
+    };
+
+    Handlebars.registerHelper('list', function(items, from, to) {
```

Comment:
Do you think that normalization will work?

The api_result looks like this
```
{"num":20,"status":"ok","r0":"http:\/\/www.myspace.com\/","r1":"https:\/\/plus.google.com\/","r2":"http:\/\/twitter.com\/", ......... }
```
Will it work if I normalize it like "r0_modifiedlink" ?